src/item.cpp

```diff
@@ -5959,7 +5959,11 @@ std::string item::display_name( unsigned int quantity ) const
             }
 
             if( max_amount != 0 ) {
-                amt = string_format( " (%i/%i%s)", amount, max_amount, ammotext );
+                const double ratio = static_cast<double>( amount ) / static_cast<double>( max_amount );
+                const nc_color color = amount == 0 ? c_dark_gray : ratio < ( 1.0 / 3.0 ) ? c_red : ratio <
+                                       ( 2.0 / 3.0 ) ? c_yellow : c_green;
```

Comment:
Could I suggest not using a ternary here for readablilty's sake?

Reply:
Ah, `astyle` didn't do that very nicely. Yeah, I'll use a good old-fashioned `if/else` here.